5.PropTypes and DefaultProps.md

```diff
@@ -0,0 +1,40 @@
+#Prop Types And Default Props
+
+##Prop Types
+PropTypes provides **typechecking ability** to react components.
+Let's take a look at some codes.
+
+```
+class Jam3Lesson extends React.Components{
```

Comment:
In code snippets always follow JS guidelines for linting. In this case, space before starting curly brace.